package.json

```diff
@@ -34,7 +34,6 @@
     "@storybook/addon-links": "^5.0.11",
     "@storybook/addon-notes": "^5.0.11",
     "@storybook/addon-viewport": "^5.0.11",
-    "@storybook/addon-a11y": "^5.0.11",
```

Comment:
why?

Reply:
@filrak this was removed because it's duplicated I think 😉 there are two lines of the same module